pkg/addons/embeddedclusteroperator/values.go

```diff
@@ -16,6 +17,10 @@ func (e *EmbeddedClusterOperator) GenerateHelmValues(ctx context.Context, kcli c
 	if err != nil {
 		return nil, errors.Wrap(err, "marshal helm values")
 	}
+
+	// replace proxy.replicated.com with the potentially customized proxy registry domain
+	marshalled = strings.ReplaceAll(marshalled, "proxy.replicated.com", e.ProxyRegistryDomain)
```

Comment:
how come you dont have to check if it is empty here but you do in the ChartLocation function?

Reply:
I should!

Reply:
done